### DISCUSSION TITLE
Buffer operator

### DISCUSSION THREAD

**Discussion by ctrlaltdan** [paillave/Etl.Net#433] [Q&A]
Hi, I was wondering if there is any support for a Buffer operator (specifically mirroring the `Buffer(timeSpan, count)` Rx operator.

Use case is to parse and load chunked output onto a service bus queue.

**Comment by ctrlaltdan** [comment]
Have just found the[ `Chunk(string name, int chunkSize)`](https://github.com/paillave/Etl.Net/blob/master/src/Paillave.Etl/Extensions/Chunk.Stream.ex.cs#L7) operator.

I think this will be sufficient, will leave this ticket open in case you'd like to opine.

**Comment by paillave** [comment]
There is no Buffer operator at the moment.
If you really need it, a custom operator shoud be doable easily. Let me know (U would have to wait!)
